### ISSUE TITLE
Training Question

### ISSUE THREAD

**Issue by iPsych** [justusschock/shapenet#33]
Hello,
I found your answer in https://github.com/justusschock/shapenet/issues/31.
Pre-trained weight of Thermal faces are still not available?
I downloaded ThermalFace Database, and about to train.
Shall I duplicate config file of helen or some adjustment is required?

**Comment by justusschock** [comment]
Hi,

No the weights are still not available and I probably won't put them online in the future as well.

I think the values of the Helen config file should be fine for the training (maybe you don't need that much of rotation).

**Comment by iPsych** [comment]
@justusschock Hi,
Thank you for the quick reply.
I am trying to train by myself using thermalface DB. however the json structure is quite different from ibug.
Did you convert it using some code or tool?

**Comment by justusschock** [comment]
no, we didn't. The thermal DB also follows the ljson standard from Menpo and was loaded using this code: https://github.com/justusschock/shapedata/blob/master/shapedata/io.py#L54 (which is also taken from Menpo/ibug)
